public/javascripts/playlistModal.js

```diff
@@ -39,6 +40,19 @@ var playlistModal = (function() {
                     args.disabled(false);
                     m.redraw()
                 }, 3000);
+
+                /*
+                  there are two different behaviors between adding to playlist from songpage and
```

Comment:
good explanation. 
Can we still show the playlists in order of recently modified so that it's convenient for the user from the song page?